components/Project.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+export default function Project({ title, duration, complexity }) {
+  return (
+    <StyledArticle>
+      <h3>{title}</h3>
+      <p>Duration: {duration}</p>
+      <p>Complexity: {complexity}</p>
+    </StyledArticle>
```

Comment:
When structuring the headings within your HTML content, it's essential to maintain a hierarchical order that reflects the content's structure and importance. 
- The `<h1>` element is typically reserved for the main title or most important heading on the page ✅ 
- Subsequent headings, `<h2>` through `<h6>`, should follow in order. 
```suggestion
    <StyledArticle>
      <h2>{title}</h2>
      <p>Duration: {duration}</p>
      <p>Complexity: {complexity}</p>
    </StyledArticle>
```

Reply:
Done